Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -800,6 +811,18 @@ namespace Multiplayer
         GetNetworkEntityManager()->HandleLocalRpcMessage(message);
     }
 
+    void NetBindComponent::HandleLocalAutonomousToAuthorityRpcMessage(NetworkEntityRpcMessage& message)
+    {
+        message.SetRpcDeliveryType(RpcDeliveryType::AutonomousToAuthority);
+        GetNetworkEntityManager()->HandleLocalRpcMessage(message);
+    }
+
+    void NetBindComponent::HandleLocalAuthorityToClientRpcMessage(NetworkEntityRpcMessage& message)
+    {
+        message.SetRpcDeliveryType(RpcDeliveryType::AuthorityToClient);
+        GetNetworkEntityManager()->HandleLocalRpcMessage(message);
+    }
```

Comment:
Unit tests exercising these would be good.

Reply:
Working on it now.

Reply:
Done.